src/code/z_parameter.c

```diff
@@ -163,50 +159,52 @@ static Gfx sScreenFillSetupDL[] = {
     gsSPEndDisplayList(),
 };
 
-s16 D_801BF9B0 = 0;
-f32 D_801BF9B4[] = {
+s16 sBButtonDoActionTextureScale = 0;
+f32 sBButtonDoActionTextureScales[] = {
+    // 100 is 1:1 scale, > 100 magnifies
     100.0f, // LANGUAGE_JPN
     109.0f, // LANGUAGE_ENG
     // Data missing for other languages?
 };
-s16 D_801BF9BC[] = {
-    0x226, // EQUIP_SLOT_B
-    0x2A8, // EQUIP_SLOT_C_LEFT
-    0x2A8, // EQUIP_SLOT_C_DOWN
-    0x2A8, // EQUIP_SLOT_C_RIGHT
+s16 sItemIconTextureScales[] = {
+    (s16)(1.074219 * (1 << 10)) >> 1, // EQUIP_SLOT_B
+    (s16)(1.328125 * (1 << 10)) >> 1, // EQUIP_SLOT_C_LEFT
+    (s16)(1.328125 * (1 << 10)) >> 1, // EQUIP_SLOT_C_DOWN
+    (s16)(1.328125 * (1 << 10)) >> 1, // EQUIP_SLOT_C_RIGHT
```

Comment:
What are these values? they seem oddly specific

Reply:
They're texture coordinate scales, which are best understood as s5.10 fixed-point numbers. The `>> 1` is present since the code that uses these does `<< 1` after loading the array value, for some reason.